### ISSUE TITLE
Microsoft Edge

### ISSUE THREAD

**Issue by onlyredwhite** [k-ivan/visual-bookmarks-chrome#63]
When I close Edge with just Visual Bookmarks the next time I open Edge it always asks whether I want to restore tabs. Normally I cancel it but if I say restore it brings up my Visual Bookmarks. So this is annoying to say the least. 

This is also the same if I close Edge by clicking the X of the Visual Bookmarks tab when it is the last tab.

Thanks - John

**Comment by k-ivan** [comment]
Hello @onlyredwhite !

If I understand correctly, then every time you close even a single tab with bookmarks, and when you open it again, the edzh offers to restore the closing of the tab?
This is very strange and looks like a local problem. Usually, the browser displays a popup with this question if its work was interrupted unexpectedly.

Perhaps I misunderstood you, in that case, could you please record a short video or describe the problem in more detail.

**Comment by onlyredwhite** [comment]
Hello,Thanks so much for the message and sorry for delay.Ok I have narrowed it down now - so I have MS Edge running and I close down with either the x on the home tab (visual book marks) or the top right hand corner X.At night I close the laptop down (after closing Edge as described above).In the morning I initiate MS Edge and it always comes up with Do You Want To Restore your tabs. Normally I click 'no' so I get up Visual Bookmarks come up which is what I want HOWEVER if I say restore the Visual Bookmarks comes up. So, this tells me that perhaps Visual Bookmarks has left something set somewhere that gets picked up MS Edge.I hope that explains it better but let me know and I'll take video of it otherwise.Thanks for reading - John-----Original Message-----From: ***@***.***Sent: Sun, 28 Mar 2021 02:03:47 -0700To: ***@***.***Subject: Re: [k-ivan/visual-bookmarks-chrome] Microsoft Edge (#63)
Hello @onlyredwhite !
If I understand correctly, then every time you close even a single tab with bookmarks, and when you open it again, the edzh offers to restore the closing of the tab?
This is very strange and looks like a local problem. Usually, the browser displays a popup with this question if its work was interrupted unexpectedly.
Perhaps I misunderstood you, in that case, could you please record a short video or describe the problem in more detail.

**Comment by k-ivan** [comment]
Hi @onlyredwhite 
Yeah, thanks for the detailed description.  It looks strange.  I doubt it is related to the extension.  I tried to reproduce myself, but there is no such behavior.  I'll try again and answer this weekend.  Thank you for your patience.

**Comment by onlyredwhite** [comment]
Hello,No problem - thanks for looking as I know there must be so many variables behind the scenes as it were. John-----Original Message-----From: ***@***.***Sent: Thu, 01 Apr 2021 10:01:15 -0700To: ***@***.***Subject: Re: [k-ivan/visual-bookmarks-chrome] Microsoft Edge (#63)
Hi @onlyredwhite
Yeah, thanks for the detailed description.  It looks strange.  I doubt it is related to the extension.  I tried to reproduce myself, but there is no such behavior.  I'll try again and answer this weekend.  Thank you for your patience.

**Comment by onlyredwhite** [comment]
Hello,FYI I remembered to do a screen shot of Edge after a machine start up this morning. visual bookmarks frame that comes up. Hope this helps a little.Thanks, John-----Original Message-----From: ***@***.***Sent: Thu, 01 Apr 2021 10:01:15 -0700To: ***@***.***Subject: Re: [k-ivan/visual-bookmarks-chrome] Microsoft Edge (#63)
Hi @onlyredwhite
Yeah, thanks for the detailed description.  It looks strange.  I doubt it is related to the extension.  I tried to reproduce myself, but there is no such behavior.  I'll try again and answer this weekend.  Thank you for your patience.

**Comment by k-ivan** [comment]
Hi, screenshot is not attached.

**Comment by onlyredwhite** [comment]
Thats strange I have looked at the email I sent and it was there OK and I've clicked it to make sure - it was only 587k so not big. I'll try again with this message. John 079 574 595 ***@***.***Original Message-----From: ***@***.***Sent: Fri, 02 Apr 2021 00:50:07 -0700To: ***@***.***Subject: Re: [k-ivan/visual-bookmarks-chrome] Microsoft Edge (#63)
Hi, screenshot is not attached.

**Comment by k-ivan** [comment]
You can attach it in the message on the github.

**Comment by onlyredwhite** [comment]
![Edge on startup](https://user-images.githubusercontent.com/81508008/113437391-73d08400-93de-11eb-867e-8049c2257468.jpg)

Hope that works k-ivan

**Comment by k-ivan** [comment]
Hi @onlyredwhite 

I wanted to know if you closed this issue because the problem is no longer relevant?

I was unable to reproduce this behavior on my own, the browser does not show the popup (everything is ok) At the moment I can only assume that the problem is local and not related to the extension itself.

**Comment by onlyredwhite** [comment]
Hello k-ivan,I did not knowingly close the issue and assumed you had!Anyways, thanks for the message. I am still getting the 'funny' after every windows start up when I enter Edge.Thanks for looking at the issue. The only thing I could try now would be:...If I go into Edge/Extensions and just click the button on the right - does it just 'turn off' so I could test that route? Then I can click 'on' again and I would have all my visual bookmarks again without me losing any? Thanks,John-----Original Message-----From: ***@***.***Sent: Mon, 19 Apr 2021 02:33:59 -0700To: ***@***.***Subject: Re: [k-ivan/visual-bookmarks-chrome] Microsoft Edge (#63)
Hi @onlyredwhite
I wanted to know if you closed this issue because the problem is no longer relevant?
I was unable to reproduce this behavior on my own, the browser does not show the popup (everything is ok) At the moment I can only assume that the problem is local and not related to the extension itself.

**Comment by k-ivan** [comment]
Turning the extension on and off or uninstalling all your bookmarks will not disappear.

I would also advise you to try resetting your browser settings to default.
Also, if other extensions are used, temporarily try to disable all extensions and check only with enabled bookmarks.

**Comment by onlyredwhite** [comment]
Hi k-ivan,I have tried a few options but in the end I turned the only 3 extensions I have and restarted then no issue with Edge.Turning Visual Bookmarks to be the ONLY extension and I return to the issue where Edge thinks it needs to restore.There is a message that comes up for a short time that says something like Visual Bookmarks will continue to work in the background. Is this a clue to the issue? The exact message is: - it is a Edge message that appears lower right that says' Visual Bookamrks will launch at system startup and continue to run in the background even once you've closed all other Microsoft Edge windows'Thanks,John 079 574 595 ***@***.***Original Message-----From: ***@***.***Sent: Mon, 19 Apr 2021 06:28:40 -0700To: ***@***.***Subject: Re: [k-ivan/visual-bookmarks-chrome] Microsoft Edge (#63)
Turning the extension on and off or uninstalling all your bookmarks will not disappear.
I would also advise you to try resetting your browser settings to default.
Also, if other extensions are used, temporarily try to disable all extensions and check only with enabled bookmarks.

**Comment by k-ivan** [comment]
Hi @onlyredwhite !
Very strange. Try to disable this option if it is enabled and check again
![Безымянный](https://user-images.githubusercontent.com/6150895/115682218-edb0b900-a35d-11eb-8695-0ffcb68bae81.png)

To be honest, there are no ideas yet.

**Comment by onlyredwhite** [comment]
Hello k-ivan,Thats good enough for me!!!I did as you suggested with the System setting and the problem has gone away. 99.9% certain that switching that option off does not affect me as a pretty standard user I would say. (I did used to be in IT just over 32 years (gcse, university, trainee programmer thru to consultant - but retired now - working with big ticket retail systems - purchasing, corporate data mgt, warehouse management for J Sainsbury, M&S, Asda etc but now just computer stuff for modern day life living!!).Thanks so much for helping me out (I did do a lot of testing over the years so once I had a 'funny' I was like a dog with a bone and just had to track issues down).Anyways enough rambling -  Best wishes - John-----Original Message-----From: ***@***.***Sent: Thu, 22 Apr 2021 01:30:23 -0700To: ***@***.***Subject: Re: [k-ivan/visual-bookmarks-chrome] Microsoft Edge (#63)

Hi @onlyredwhite !

Very strange. Try to disable this option if it is enabled and check again



To be honest, there are no ideas yet.
